pysam/libcalignedsegment.pyx

```diff
@@ -2448,8 +2442,9 @@ cdef class PileupColumn:
             # warning: there could be problems if self.n and self.buf are
             # out of sync.
             for x from 0 <= x < self.n_pu:
-                pileups.append(makePileupRead(&(self.plp[0][x]),
-                                              self._alignment_file))
+                pileups.append(
```

Comment:
I'm having trouble justifying why 79/80 column limits are still relevant in this day and age.  My laptop display can show 150+ columns with my default font.  Can we decide to defy PEP8 a little and support a 120 column limit in pysam?